tempesta_fw/http_sess.c

```diff
@@ -565,7 +566,7 @@ tfw_http_sess_obtain(TfwHttpReq *req)
 			goto found;
 	}
 
-	if (!(sess = kmem_cache_alloc(sess_cache, GFP_ATOMIC))) {
+	if (!(sess = kmem_cache_zalloc(sess_cache, GFP_ATOMIC))) {
```

Comment:
Don't use __GFP_ZERO w/o need, that's relatively expensive thing to make the kernel write zeroes to free pages. Instead, you can just write 3 NULLs  to the new pointer fields.